src/Build/project.json

```diff
@@ -1,4 +1,7 @@
 ﻿{
+  "dependencies": {
+    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0"
```

Comment:
Be sure to add the assembly to our custom copy logic here: https://github.com/cdmihai/msbuild/blob/fa2c0f8541ed96a303927131abe268952bf37706/src/Build/Microsoft.Build.csproj#L798

You might need to also add it to the build definition so that it ends up in our VSIX.